inst/schema/InputTimeSeriesResponse.schema.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "data": { "$ref": "InputTimeSeriesData.schema.json" },
+    "plottingMetadata": {
+      "type": "object",
+      "properties": {
+        "barchart": { "$ref": "BarchartMetadata.schema.json" }
```

Comment:
I guess it's good to re-use the BarchartMetadata schema here, but I don't think the prop should be called 'barchart' since the charting implementation is going to be different from the old charts - and it won't necessarily be a barchart. 
I wouldn't mind if 'plottingMetadata' just became 'metadata' and that was a BarchartMetadata object, or could stick with 'plottingMetadata' and replace 'barchart' with 'genericChart'..?

Reply:
Oops - that is complete nonsense left from a copy + pasting. Thanks for catch